soccer/src/soccer/strategy/coach/coach_node.hpp

```diff
@@ -51,6 +60,15 @@ class CoachNode : public rclcpp::Node {
     void world_state_callback(const rj_msgs::msg::WorldState::SharedPtr msg);
     void ball_sense_callback(const rj_msgs::msg::RobotStatus::SharedPtr msg, bool our_team);
     void check_for_play_state_change();
+    /*
+     * Calls assign_positions and check_for_play_state_change in coach_change_timer
```

Comment:
I know that's what is happening now, but in the future we could add more things we want the coach to do every tick. So I would lead with something like "Handle actions the Coach does every tick (see <timer-name>). Currently calls [...]" and then list what you have there

Reply:
also I think the name `coach_change_timer` is a bit confusing, because sometimes nothing changes but the timer will still tick, right?

Reply:
Yea I agree with the name of the timer is bad. I'll change the docstrings for it as well. I think I'll call it coach_actions_timer.